js/binance.js

```diff
@@ -400,6 +412,12 @@ module.exports = class binance extends Exchange {
                         //
                         'giftcard/createCode': 0.1,
                         'giftcard/redeemCode': 0.1,
+                        'algo/futures/newOrderVp': 300,
```

Comment:
Which formula do you use to get this number 300 ?

If you check this endpoint, it says WEIGHT UID 3000:

- https://binance-docs.github.io/apidocs/spot/en/#volume-participation-vp-new-order-trade

<img width="1545" alt="Screenshot 2022-05-01 at 21 47 26" src="https://user-images.githubusercontent.com/1294454/166160113-d3a7d549-7f32-4536-9823-46e24ac7fd07.png">

If you check the other endpoints above, you will note that WEIGHT UID 3000 yields a cost of 20:

<img width="844" alt="Screenshot 2022-05-01 at 21 48 56" src="https://user-images.githubusercontent.com/1294454/166160172-bd054648-23b9-4d56-aec5-b32c26de6d64.png">

So, I guess, it should have a weight of 20 or 20.001 as well ?

Please, check the other endpoints to make sure that a correct rateLimit value is used. Thx!